src/Shared/ProgramArguments.Shared.cs

```diff
@@ -387,6 +392,37 @@ public bool TryGetEntryProjectPaths(ISlnGenLogger logger, out IReadOnlyList<stri
             return result.Count > 0;
         }
 
+        private static IEnumerable<string> ExpandWildcards(IEnumerable<string> paths)
```

Comment:
Please add a unit test for this method to ensure it doesn't get regressed

Reply:
Done